mutationanalysis.properties

```diff
@@ -1,12 +1,18 @@
 category=Coverage
 description=Import Pitest mutation analysis results
 homepageUrl=https://github.com/devcon5io/mutation-analysis-plugin
-archivedVersions=1.2.1,1.3
-publicVersions=1.4
+archivedVersions=1.2.1,1.3,1.4
+publicVersions=1.5
 
 defaults.mavenGroupId=ch.devcon5.sonar
 defaults.mavenArtifactId=mutation-analysis-plugin
 
+1.5.description=Support for Surviving Mutants as Code Smells
+1.5.sqVersions=[6.7,LATEST]
+1.5.date=2019-08-22
+1.5.changelogUrl=https://github.com/devcon5io/mutation-analysis-plugin/releases/tag/v1.5
+1.5.downloadUrl=https://github.com/devcon5io/mutation-analysis-plugin/releases/download/v1.5/mutation-analysis-plugin-1.5.jar
+
 1.4.description=Gradle support, new metric for numbers of tests run per mutant
 1.4.sqVersions=[6.7,LATEST]
```

Comment:
Only one version can be compatible with `LATEST`. The convention is to update the end range of the version being archived to the version of SQ that is current at the time.

Reply:
I updated to the LATEST to 7.9 for the previous version (1.4), I hope that's correct?